quickfixj-core/src/main/doc/usermanual/usage/configuration.html

```diff
@@ -601,6 +601,13 @@ <H3>QuickFIX Settings</H3>
     </TD>
     <TD> empty, ie all remote addresses are allowed </TD>
   </TR>
+  <TR ALIGN="left" VALIGN="middle">
+    <TD valign="top"> <I>LogMessageWhenSessionNotFound</I></TD>
```

Comment:
I think this should be moved to the Misc section since it is applicable to Acceptors as well as Initiators. Sorry I was not more specific when requesting this.